core/src/commonMain/resources/MR/base/strings.xml

```diff
@@ -4,6 +4,10 @@
     <!--suppress CheckTagEmptyBody -->
     <string name="blank" translatable="false"></string>
     <string name="login">Login</string>
+    <string name="share_app">Share app</string>
+    <string name="share_this_app_with_others">Share This App with Others</string>
```

Comment:
This string (share_this_app_with_others) is no longer used, like anything else that is unused, it should be removed.